lib/sitemap_generator.rb

```diff
@@ -54,9 +54,10 @@ def reset!
   end
 
   class << self
-    attr_accessor :root, :app, :templates
+    attr_accessor :default_changefreq, :root, :app, :templates
     attr_writer :yield_sitemap, :verbose
   end
+  @default_changefreq = 'monthly'
```

Comment:
Would `weekly` be a better default?  I'm not certain `monthly` is best, but I'm also not certain it matters very much 🤷